### PR TITLE
Owl implementation

### PR DESCRIPTION
**Pull request by sammurraytuesta**
Separate PR for the implementation of the owl carousel. Stems from the main branch rather than from design-edits branch.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```
Comment (rjkroege) [inline portfolio/pom.xml]: This seems out of scope for this PR.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,19 +1,107 @@
...
-  </body>
-</html>
+
+    <!-- owl carousel -->
+    <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+    <script src="owl.carousel/dist/owl.carousel.min.js"></script>
+    <script src="script.js"></script>
+</body>
+
+</html>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html:226]: Need a trailing newline.

File: portfolio/src/main/webapp/owl.carousel/.gitattributes [hunk 3]
```diff
@@ -0,0 +1,5 @@
+* text eol=lf
```
Comment (rjkroege) [inline portfolio/src/main/webapp/owl.carousel/.gitattributes]: This file should be in the "add Owl" PR, not this one.

**Review by rjkroege — CHANGES_REQUESTED** [review]
I don't think that you have separated your use of Owl from the addition of the Owl library. This PR should not also add OWL right?

File: portfolio/pom.xml (hunk 1 above)
Comment (sammurraytuesta) [reply portfolio/pom.xml]: Sorry about that

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html:226]: okay thank you

File: portfolio/src/main/webapp/owl.carousel/.gitattributes (hunk 3 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/owl.carousel/.gitattributes]: it is also in the add Owl PR, I believe
